src/utils/user_utils.js

```diff
@@ -21,16 +22,14 @@ export function displayUsername(
     user: UserProfile,
     teammateNameDisplay: Preferences.DISPLAY_PREFER_NICKNAME | Preferences.DISPLAY_PREFER_FULL_NAME
 ): string {
-    let name = '';
+    let name = localizeMessage('channel_loader.someone', 'Someone');
 
     if (user) {
         if (teammateNameDisplay === Preferences.DISPLAY_PREFER_NICKNAME) {
             name = user.nickname || getFullName(user);
         } else if (teammateNameDisplay === Preferences.DISPLAY_PREFER_FULL_NAME) {
             name = getFullName(user);
-        }
```

Comment:
Do we still want the `name.trim().length` check, to keep "Someone" in the event the name really does only contain whitespace? Not sure if that's really possible or not, so it might be fine as-is.

Reply:
Trim was used for detecting empty default string if  `teammateNameDisplay` does not satisfy both the preference settings. This was added in the very first commit of the lib so no history to traceback here. Before removing trim and moving it to else my UI broke to 
![screen shot 2018-05-02 at 11 21 51 pm](https://user-images.githubusercontent.com/4973621/39577523-3541fda8-4eff-11e8-85d0-f445fa5ff090.png) as the default string was important for returning displayNames in most scenarios.

We could trim but it is odd doing it only in one condtion and not everywhere as this could happen for firstname and lastname as well.